welder-web.spec

```diff
@@ -0,0 +1,25 @@
+Name: welder-web
+Version: 0
+Release: 0
```

Comment:
Okay. I am looking into how to do this in a manner compatible with copr. You want it to set the version based on the version in the package.json correct? Hopefully I'll have a fix sometime this week.